dotnet/src/SemanticKernel/IKernel.cs

```diff
@@ -156,4 +158,14 @@ public interface IKernel
     /// </summary>
     /// <returns>SK context</returns>
     SKContext CreateNewContext();
+
+    /// <summary>
```

Comment:
Wondering if this makes more sense on KernelConfig -- that's where I'm adding the Http DelegatingHandler Factory -- thoughts? If this makes more sense, then can refactor that factory here for consistency.